manifests/integrations/postgres.pp

```diff
@@ -19,6 +19,12 @@
 #       Track per relation/table metrics. Array of strings.
 #       Warning: this can collect lots of metrics per relation
 #       (10 + 10 per index)
+#   $tags
+#       Optional array of tags
+#   $custom_metrics
+#       An array of custom metrics with the following hash keys - query, metrics,
+#       relation, descriptors. Refer to this guide for details on those fields:
+#       https://help.datadoghq.com/hc/en-us/articles/208385813-Postgres-custom-metric-collection-explained
```

Comment:
Could you please add an example with custom metrics below? Should help a bit with usability.